packages/client-config/API.md

```diff
@@ -208,7 +208,7 @@ export type CustomClientConfig = {
 export const DEFAULT_CLIENT_CONFIG_VERSION: ClientConfigVersion;
 
 // @public
-export const generateClientConfig: <T extends "1" | "0">(backendIdentifier: DeployedBackendIdentifier, version: T, awsClientProvider?: AWSClientProvider<{
+export const generateClientConfig: <T extends "0" | "1">(backendIdentifier: DeployedBackendIdentifier, version: T, awsClientProvider?: AWSClientProvider<{

```

Comment:
PR checks might complain about this. If they do please revert this change.
(for some reason its being non deterministic between platforms/OSes).

Reply:
Reverted this file